keanu-project/src/main/java/io/improbable/keanu/vertices/VertexId.java

```diff
@@ -80,6 +87,12 @@ public int getIndentation() {
         return idValues.length;
     }
 
+    public KeanuSavedBayesNet.VertexID toProtoBuf() {
```

Comment:
`toProtoBuf<T>` looks like a good candidate for an interface

Reply:
100% - good suggestion.  I'll get this updated.